_data/investing.yml

```diff
@@ -98,8 +98,11 @@ websites:
     - name: Nutmeg
       url: https://www.nutmeg.com/
       twitter: thenutmegteam
+      facebook: TheNutmegTeam
```

Comment:
'twitter' should NOT be present when `tfa: true`.
'facebook' should NOT be present when `tfa: true`.